cmd/proxy/main.go

```diff
@@ -44,6 +44,20 @@ func main() {
 
 	logger := athenslog.New(conf.CloudRuntime, logLvl)
 
+	// logrus.Writer() spawns a go routine, so instead of doing that each time
+	// we need to make logrus look like a (standard) log.Logger (and then Close()ing it),
+	// we can do it once in main and have the global log.Logger use it.
+	{
+		logrusWriter := logger.WriterLevel(logrus.ErrorLevel)
+		defer func() {
+			if err := logrusWriter.Close(); err != nil {
+				logger.WithError(err).Warn("Could not close logrus writer pipe")
+			}
+		}()
+		stdlog.SetOutput(logrusWriter)
+		stdlog.SetFlags(stdlog.Flags() &^ (stdlog.Ldate | stdlog.Ltime))
+	}
```

Comment:
Scoping the variables here seems unnecessary, unless I am missing something that isn't described by the comment.

```suggestion
	logrusWriter := logger.WriterLevel(logrus.ErrorLevel)
	defer func() {
		if err := logrusWriter.Close(); err != nil {
			logger.WithError(err).Warn("Could not close logrus writer pipe")
		}
	}()
	stdlog.SetOutput(logrusWriter)
	stdlog.SetFlags(stdlog.Flags() &^ (stdlog.Ldate | stdlog.Ltime))
```

WDYT of simply removing the scope?

Reply:
I did it just to logically group things together, and also to discourage/prevent using `logrusWriter` later on. I'm fine removing it if this feels too awkward/unusual to others.

Reply:
Removed scope in 6eea0a6.